### PR TITLE
Add README file

### PR DESCRIPTION
**Pull request by crojewsk-intel**
Provide overview segment and contribution guidelines for the project.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+Converts topology configuration files from a more friendly XML format into UCM i.e.: alsa-lib
+friendly format.
+
+[avs-topology-xml](https://github.com/thesofproject/avs-topology-xml) hosts a wide range of topology
+configuration files which are input for this tool.
+
+## fwlog_parser
+
+Parses the binary AudioDSP log file into a human-friendly text equivalent given the symbol
+dicitonary. The symbol dictionaries are not part of the repo yet. References will be provided in the
```
Comment (amadeuszslawinski-intel) [inline README.md]: s/dicitonary/dictionary
s/repo/repository/

File: README.md [hunk 2]
```diff
@@ -76,3 +81,58 @@ to given module.
...
+*   The lower level the problem is, the more detailed the description
+    should be.
+*   Ensure any links or references appended are actually working
+*   Keep the line width with 72 characters margin
+
+## Commit tags
+
+Optionally, at the bottom of commit message tags (kind of keywords) can be appended. These tags
+serve informational purposes, further enhancing code documentation and enabling fixes propagation
+through automated tasks. Below are guides for a range of such tags:
```
Comment (amadeuszslawinski-intel) [inline README.md:132]: Should mention that Signed-of is mandatory here?

Comment (crojewsk-intel) [reply README.md:132]: Excellent finding!

File: README.md (hunk 1 above)
Comment (crojewsk-intel) [reply README.md]: Ack.

**Review by amadeuszslawinski-intel — APPROVED** [review]

**Pull request merged**